examples/README.md

```diff
@@ -23,49 +23,59 @@ trunk serve --release
 We're also publicly hosting the examples at `https://examples.yew.rs/<EXAMPLE>`.
 As an example, check out the TodoMVC example here: <https://examples.yew.rs/todomvc>
 
-## List of examples
+## List of Examples
 
- - CT - Type of most components , "F" for function components and "S" for struct components, "SF" for a mix of both.
+- `Example`: Link and name of each example.
+- `CT`: Type of most components
```

Comment:
```suggestion
- `CT`: Component type of most components
```

I'm not a big fan of this legend style. Maybe we can get away with [the "usual" hack](https://github.com/BoostIO/BoostNote-Legacy/issues/3579#issuecomment-855449583) of fake links that show tooltips when hovered? Would look like this

```
|   | [CT] |    |
|---|------|----|
|   | [S]  |    |
|   | [F]  |    |
|   | [SF] |    |

[CT]: ## "Component type"
[S]: ## "Struct components"
[F]: ## "Function components"
[SF]: ## "Mixes struct and function components"
```

Renders as

|   | [CT](## "Component type") |    |
|---|-----------------|----|
|    | [S] |    |
|    | [F] |    |
|    | [SF] |    |

[S]: ## "Struct components"
[F]: ## "Function components"
[SF]: ## "Mixes struct and function components"

Reply:
I'll add this in now.